runtime/gc_modron_startup/mgcalloc.cpp

```diff
@@ -363,10 +363,10 @@ J9AllocateIndexableObjectNoGC(J9VMThread *vmThread, J9Class *clazz, uint32_t num
 				uintptr_t allocatedBytes = env->getExtensions()->objectModel.getConsumedSizeInBytesWithHeader(objectPtr);
 
 				/* Do sanity check: size of actually allocated Indexable object should match requested */
-				uintptr_t actuallyAllocatedBytes = indexableOAM.getAllocateDescription()->getContiguousBytes();
-				Assert_GC_true_with_message4(env, allocatedBytes == actuallyAllocatedBytes,
-						"Indexable object allocation sanity failure: object %p, requested %zu bytes, but read %zu, MM_IndexableObjectAllocationModel %p\n",
-						objectPtr, allocatedBytes, actuallyAllocatedBytes, &indexableOAM);
+				uintptr_t requestedBytes = indexableOAM.getAllocateDescription()->getContiguousBytes();
+				Assert_GC_true_with_message4(env, allocatedBytes == requestedBytes,
+						"J9AllocateIndexableObjectNoGC: Indexable object allocation sanity failure: object %p, requested %zu bytes, but read %zu, MM_IndexableObjectAllocationModel %p\n",
```

Comment:
perhaps you can reduce the size of the message and remove 'Indexable object allocation' part
it's already stated in J9AllocateIndexableObjectNoGC and MM_IndexableObjectAllocationModel that it's about allocation and about an indexable object
same for mixed...